src/components/ThoughtAnnotation.tsx

```diff
@@ -197,7 +208,11 @@ const ThoughtAnnotation = React.memo(
             //   border-bottom: solid 1px;
             // }
           }
-          style={styleAnnotation}
+          style={{
+            padding: `0 ${thoughtAnnotationHorizontalPadding}px`,
```

Comment:
I'm concerned that `paddingRight` (above) will be overridden now that `padding` has been moved to `style`. It might be a good idea to move `paddingRight` with it.